styles/reset.css

```diff
@@ -49,3 +49,7 @@ h6 {
 #__next {
   isolation: isolate;
 }
+
+ul {
+  list-style: none;
+}
```

Comment:
Removing the list-style breaks accessibility on unordered lists for screen readers. What you could do instead is:

```
ul[role="list"],
ol[role="list"] {
  list-style: none;
}
```
This will remove visual styles on lists when you place a `role="list"` on your element, but maintain the list semantics for accessibility.